scripts/2.mutation-classifier.py

```diff
@@ -181,31 +188,30 @@ def select_feature_set_columns(X, feature_set):
                 for mod, pipeline in pipeline_definitions.items()}
 
 
-# In[14]:
-
-st = time.perf_counter()
+# In[13]:
 
 # Fit the models
 for model, pipeline in cv_pipelines.items():
     print('Fitting CV for model: {0}'.format(model))
+    start_time = time.perf_counter()
     pipeline.fit(X=X_train, y=y_train)
-    
-et = time.perf_counter()
-print('Time to fit models: {0}'.format(str(datetime.timedelta(seconds=et-st))))
+    end_time = time.perf_counter()
+    elapsed = datetime.timedelta(seconds=end_time - start_time)
+    print('\truntime: {}'.format(elapsed))
 
 
-# In[15]:
+# In[14]:
 
 # Best Parameters
 for model, pipeline in cv_pipelines.items():
-    print('{0}: {1:.3%}'.format(model, pipeline.best_score_))
-
+    print('#', model)
     print(pipeline.best_params_)
+    print('cv_auroc = {:.3%}'.format(pipeline.best_score_))
 
```

Comment:
Makes sense to me. I think `StratifiedKFold` will be especially useful for unbalanced genes. I'm in favor of option 2 as well.

Regarding reusing CV folds. I'm not sure if you were talking about using the same folds for each model (expression/covariates/full) or the same folds for each parameter combination within an individual model. If you were referring to the later...  I think this is one of the ways `dasksearchcv` speeds up gridsearch. 

On the topic of `dasksearchcv`: I'll start the `dasksearchcv` PR after this PR is merged.